tests/charm-secret/pyproject.toml

```diff
@@ -18,7 +18,7 @@ target-version = ["py38"]
 [tool.ruff]
 line-length = 99
 select = ["E", "W", "F", "C", "N", "D", "I001"]
-extend-ignore = [
+lint.extend-ignore = [
```

Comment:
Apparently that too is deprecated, and it's just "ignore" now,,,
Totally OK to leave to a separate PR.
https://docs.astral.sh/ruff/settings/#lint_extend-ignore